pallets/author-mapping/src/lib.rs

```diff
@@ -96,16 +117,18 @@ pub mod pallet {
 		AuthorRegistered {
 			author_id: NimbusId,
 			account_id: T::AccountId,
+			keys: T::Keys,
 		},
 		/// An NimbusId has been de-registered, and its AccountId mapping removed.
-		AuthorDeRegistered { author_id: NimbusId },
+		AuthorDeRegistered { author_id: NimbusId, keys: T::Keys },
 		/// An NimbusId has been registered, replacing a previous registration and its mapping.
-		AuthorRotated {
+		KeysRotated {
 			new_author_id: NimbusId,
+			new_keys: T::Keys,
 			account_id: T::AccountId,
 		},
 		/// An NimbusId has been forcibly deregistered after not being rotated or cleaned up.
-		/// The reporteing account has been rewarded accordingly.
+		/// The reporting account has been rewarded accordingly.
 		DefunctAuthorBusted {
```

Comment:
I checked and we never use this event in our codebase, I think it is better to delete it.

Reply:
done